### PR TITLE
Credits a bunch of people for clock cult

### PR DESCRIPTION
**Pull request by ghost**
@VelardAmakar
@SkowronX
@FuryMcFlurry
@PJB3005
@ChangelingRain

### CODE REVIEW

**Comment by yackemflam** [comment]
You really shouldn't put names in the code.

**Comment by Xhuis** [comment]
There's tons of names in our code. These are comments and don't affect the game in any way.

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -27,6 +27,14 @@ This file's folder contains:
 	clock_unsorted.dm: Anything else with no place to be
 
 	clockcult defines are in __DEFINES/clockcult.dm
+
+Credit where due:
+1. SkowronX from /vg/ for the original idea and code
+2. FuryMcFlurry from /vg/ for many of the assets
+3. PJB3005 from /vg/ for the failed continuation PR
```
Comment (PJB3005) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: *AHEM.*

I have simply shifted priorities to more pressing matters.

ONE DAY I WILL MAKE /vg/ CLOCKCULT.

IT'S GONNA BE GLORIOUS.

JUST YOU WATCH.

**Comment by RemieRichards** [comment]
casually editing Velard out of history?

**Comment by Xhuis** [comment]
Who? Actually serious, I don't know who that is. I'll credit him if he's important.

File: code/game/gamemodes/clock_cult/clock_cult.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: But why

**Comment by RemieRichards** [comment]
Amazing really.
Clockcult is Velard's gamemode, they're the one who wrote all the cool design docs and lore, which atm you've accredited to Skowron, Skowron just did lots of (admittedly AWESOME) spritework for clockcult.

**Comment by Xhuis** [comment]
I had no idea, thank you. I'll throw that in right away.

**Comment by optimumtact** [comment]
why should we credit them, Xhuis did all the real work.

**Comment by Xhuis** [comment]
I may have done all of the code, but sprites and the original idea and design document were almost entirely created by /vg/. They deserve the credit for their idea.

**Comment by optimumtact** [comment]
Credit goes to those who DO!

**Comment by Jordie0608** [comment]
I'm un-fond of the 30 line comment in this file.

**Comment by Xhuis** [comment]
To be fair, @Jordie0608, this is an entire gamemode. I wanted to have a lot of documentation on what the gamemode is, what the antagonists do, and their goals. A large comment is the best way to do that.

**Comment by FuryMcFlurry** [comment]
I'm glad you got this sorted out, thanks.

**Comment by Xhuis** [comment]
@InsaneHyena Why would you downrate this lmao
Anyways, this is merge-ready

**Pull request merged**